multidim-interop/dockerBuildWrapper.sh

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env /bin/bash
 
-# If in CI use caching
-if [[ -n "${CI}" ]]; then
+# If in CI and we have a defined cache bucket, use caching
+if [[ -n "${CI}" ]] && [[ -n "${AWS_BUCKET}" ]]; then
```

Comment:
Why CI? Why not also locally?

Reply:
CI is the ephemeral runners. Locally you'll already have access to your local cache.

I don't want to deal with getting folks access to this cache bucket right now.